### ISSUE TITLE
Can not access an emulator on the web

### ISSUE THREAD

**Issue by HoonHeeLee** [google/android-emulator-container-scripts#375]
I am trying to install multiple emulators using envoy and nginx on the web in my linux box.
- https://source.android.com/docs/automotive/start/avd/cloud_emulator?hl=ko#start-containers

I could access below localhosts and could see the emulators on the web browser.
- http://localhost:19001
- http://localhost:19002

Unfortunately, external users could not access the web (envoy proxy).
![image](https://github.com/google/android-emulator-container-scripts/assets/7607717/89a2c503-5dd7-4078-bc02-b0e9af9b47ad)
  
The requested domain or ip may need to be registered to firebase authorization list.
But, I don't know how to cope this symptom.
  
Thanks.

**Comment by hridhar** [comment]
is this issue solved?
Please confirm.

**Comment by quythai-personify** [comment]
Step to resolve this:
- Create your own Firebase project
- Add your domain as OAuth redirect domain
- Generate firebase_config.json with your own firebase project information

**Comment by jpcottin** [comment]
@HoonHeeLee — the underlying problem here is now properly addressed.

The error was Firebase rejecting sign-in from a non-localhost origin, because the bundled `firebase_config.json` pointed at `android-emulator-webrtc-demo` — a Firebase project nobody on the team has had access to for some time, so its authorized-domains list couldn't be updated.

After #408 + #409 (merged today), the `js/` setup is split into two clearly documented auth paths:

- [**Self-hosted JWT**](https://github.com/google/android-emulator-container-scripts/blob/master/js/README.md#self-hosted-jwt-recommended-for-deployments) — `js/jwt-provider/` + `js/docker/`. No external identity provider, no domain registration, no Firebase. Recommended for any real deployment, including your multi-emulator scenario behind envoy on a real-IP linux box.
- [**Firebase / Google sign-in**](https://github.com/google/android-emulator-container-scripts/blob/master/js/README.md#firebase--google-sign-in-quick-start-demo-only) — quick-start demo only. You bring your own Firebase project and add your real IP/domain to its authorized list.

For your stated use case the JWT path is the better fit — it's exactly the "disable the firebase authorization function" outcome you asked for, without losing access control.

Closing as the immediate cause (dead shared Firebase project) is resolved. If you run into issues following the JWT setup, please open a new issue — the code and instructions have changed substantially since this was filed, so a fresh thread will be easier to track.
